ShopLens/ImageRecognition/TfImageUtil.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Author: Miguel de Icaza
+// Taken and adapted from: https://github.com/migueldeicaza/TensorFlowSharp/blob/master/Examples/ExampleCommon/ImageUtil.cs
```

Comment:
Maybe we should add this library and credit the author on our master branch README.md?